src/utils/typesColor.ts

```diff
@@ -0,0 +1,57 @@
+export function getComplementaryColor(type: string): string {
```

Comment:
we should give some more thought while naming some files .. in this case the utils file holds function which return based on some factor and typesColor doesn't seem to make sense here .. to me it looks like it would be a type file which would hold color interface or type.. so we would have to name this according to its purpose.  

For eg: colorUtils would also fit this .. but if have anything else you can do so which fits this file